empress/recon_vis/utils.py

```diff
@@ -87,7 +87,7 @@ def _find_roots(old_recon_graph) -> List[MappingNode]:
             roots.append(mapping)
     return roots
 
-def dict_to_reconciliation(old_recon: Dict[Tuple, List]):
+def dict_to_reconciliation(old_recon: Dict[Tuple, List], event_scores = None):
```

Comment:
You might also want to modify `dict_to_recongraph(...)` to also receive `event_scores`

Reply:
does this still exist?